doc/howto/images_remote.md

```diff
@@ -1,125 +1,73 @@
 (images-remote)=
 # How to use remote images
 
-- Link to {ref}`remote-image-servers`
-- List remotes
-- Add remotes
-- Set a default remote
-- Reference an image (`<remote>:<image>`)
-- Search for images
-- Configure caching
+LXD is pre-configured with several remote image servers.
```

Comment:
Technically speaking the `lxc` CLI command is preconfigured with several remote image servers.
LXD itself has no concept of that.

Maybe we can say this here for accuracy, something like:

> The LXC CLI command is...